charts/catalog/values.yaml

```diff
@@ -9,6 +9,7 @@ useAggregator: true
 ## If true, create & use RBAC resources
 ##
 rbacEnable: true
+rbacApiVersion: v1
```

Comment:
should be `rbac.authorization.k8s.io/v1`

Reply:
fixed